### ISSUE TITLE
Error while uploading: [c9abbb2c] ApiException

### ISSUE THREAD

**Issue by SolarSistim** [afterlogic/aurora-files#28]
Hi Afterlogic! Love the app! Hoping you can help me w/ an error during upload of larger files. Here's the deets:

Full error message: 
[13:58:52.19][bd45cf8d] [POST(4)][ip:192.168.1.141] /?/Api/
[13:58:52.19][bd45cf8d] POST > [Module, Method, Parameters, TenantName]
[13:58:52.19][bd45cf8d]
[13:58:52.19][bd45cf8d] ApiException

Files working Vs. Files NOT working:
Working: 
IMG--0317.JPG 400Kb
iji15.dll 364Kb
201542845632841.pdf 48Kb
msdia80.dll 900Kb

Errored out:
project1.psd 18Mb
battle.net.dll 15Mb

Settings:
Files - UNCHECKED: Enable file upload size limit
Personal files - 4000Mb
USer space limit: 4000Mb
Corporate space limit: 100Mb

Environment:
Windows 10 LTSC
IIS web server
Aurora on port 301
Caddy reverse proxy -> port 301

Please help. Thank you!

**Comment by afterlogic-support** [comment]
PHP configuration limits would be the first thing to check. Create a .php file in Aurora Files root directory, with the content like this:
```php
<?php phpinfo();
```
and look for `upload_max_filesize` and `post_max_size` values:

[Error while uploading to Files storage](https://afterlogic.com/docs/aurora-files/troubleshooting/error-while-uploading-to-files-storage)

**Comment by SolarSistim** [comment]
upload_max_size: 2000mb
post_max_size: 8mb <- Ah hah!

I'm going to adjust a report back
Also thank for responding 28 seconds after I posted lol!

**Comment by SolarSistim** [comment]
Problem solved! Thanks so much!

**Comment by SolarSistim** [comment]
FYI I ran into the same issue with larger files. This is not a problem with Aurora Files - it's a problem w/ IIS. I am posting this for posterity.

IIS -> Click the website in question -> click Request Filtering -> click the Headers tab -> click Add Header -> Configure this screen using the instructions below:

In the “Maximum allowed content length (Bytes)” field, enter this number, or something much larger than what is already in there:

2147483647

That fixed the upload size issue for me.

Some back story: When I originally posted this, I had ruled out IIS being the problem, stating that I had isolated the problem to Caddy - in fact, the large upload worked fine while no SSL was engaged. Once the SSL engaged, IIS began presenting this problem, and since I get my SSLs from Caddy, I wrongly assumed the problem was with Caddy.

I think the maximum header size in IIS is 4294967295, which I think is dead-on 4 Gb, and works with my configuration. I just uploaded a 3.6 Gb CD image and it worked as expected.
